aptos-labs/aptos-core
# This re-brands the `aptos-sf-stream` into `aptos-fh-stream` (`fh` as a shortcut of Firehose)

### Description

The package name and all files related to this functionality has been renamed from `sf` to `fh`.

Also, the top-level config element has been renamed from `sf_stream` to `firehose_stream` to make easier for new user to discover the functionality as `sf` or `fh` as little meaning and is hard to search on Google while at least Firehose should return some results.

Changed also some reference from `StreamingFast` to `StreamingFast Firehose` in some comments as well as fixing the `Streaming Fast` company name to be `StreamingFast`.

### Test Plan
- Build `aptos-node`, follow instructions in `fh-stream/README.md` about how to run it
- Ensure it's streaming firehose log to stdout


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/aptos-labs/aptos-core/3008)
